README.md

```diff
@@ -1 +1,263 @@
+To do this, we need to specify *what* attributes we want to be configured by. Using our Twitter card above, the only thing we need to configure it is the Twitter handle.
+
+To grab this value out of the attributes, we can specify the attribute name in the object. For instance, if we wanted to utilise the Twitter card as such:
+
+```html
+<twitter-card handle="billgates"></twitter-card>
+<twitter-card handle="bob"></twitter-card>
+```
+
+We'd put the property `handle` into our scope object.
+
+Now, what do we put as the value? There are two that we can use:
+
+#### @ (at)
+
+If we pass through `@` as the value, as so:
+
+```js
+function TwitterCard() {
+	return {
+		template: [
+			'<div class="twitter">',
+				'<a href="https://twitter.com/{{ handle }}">Follow @{{ handle }} on Twitter!</a>',
+			'</div>'
+		].join(''),
+		scope: {
+			handle: '@'
+		},
+		restrict: 'E'
+	};
+}
+
+angular
+	.module('app')
+	.directive('twitterCard', TwitterCard);
+```
+
+What `@` tells Angular is to just copy the value as it is. This will literally copy whatever we put in the `handle` attribute and put it into our scope.
+
+#### = (equals)
+
+However, if we want to be able to receive variables through, we can use `=`.
+
+```js
+function TwitterCard() {
+	return {
+		template: [
+			'<div class="twitter">',
+				'<a href="https://twitter.com/{{ handle }}">Follow @{{ handle }} on Twitter!</a>',
+			'</div>'
+		].join(''),
+		scope: {
+			handle: '='
+		},
+		restrict: 'E'
+	};
+}
+
+angular
+	.module('app')
+	.directive('twitterCard', TwitterCard);
+```
+
+What this means is that if we have a controller with two properties on it, such as `twitterHandle1` and `twitterHandle2`, we can do this:
+
+```html
+<twitter-card handle="twitterHandle1"></twitter-card>
+<twitter-card handle="twitterHandle2"></twitter-card>
+```
+
+Angular will then change the values over to their actual scope values before passing the value through to our directive. This also means that if we were to update either of them, Angular will automatically update our directive to match it too. Awesome!
+
+#### Changing attribute names
+
+Notice how we've updated our template to use `{{ handle }}`? This is because our `$scope` has the property `handle` on it now, because of the object.
+
+However, you might not like this - you might have wanted to stick with `{{ twitter }}` but *also* have it configured by using the attribute `handle`. To do this, we can change the object property to `twitter` and we put `handle` after the `@` or `=``, like follows:
+
+```js
+function TwitterCard() {
+	return {
+		template: [
+			'<div class="twitter">',
+				'<a href="https://twitter.com/{{ twitter }}">Follow @{{ twitter }} on Twitter!</a>',
+			'</div>'
+		].join(''),
+		scope: {
+            twitter: '@handle'
+        },
+		restrict: 'E'
+	};
+}
+
+angular
+	.module('app')
+	.directive('twitterCard', TwitterCard);
+```
+
+This means we can do this still:
+
+```html
+<twitter-card handle="billgates"></twitter-card>
```

Comment:
I'm confused by this example - any particular reason why you woudn't want the property to match?